src/fielddef.ts

```diff
@@ -156,6 +156,8 @@ export function field(fieldDef: FieldDef, opt: FieldRefOption = {}) {
 
   if (opt.datum) {
     field = `datum["${field}"]`;
+  } else if (field !== undefined && field.indexOf('.') !== -1) { // dot in fieldname
```

Comment:
I wonder if we should just leave as is and tell users to use `[a.a]`
if they want dot in the name and then use `a.a` for nested JSON